### DISCUSSION TITLE
<web> GLB file loads locally but not on my website (only in the parent folder)

### DISCUSSION THREAD

**Discussion by renanmalato** [google/model-viewer#4644] [Q&A]
Hi guys,

someone knows why my GLB file loads on my website only if I call it in a parent folder, like `src="folder/file.glb"`? 

If i put in my website in another folder and call the `src="www.mywebsite.com/folder/folder/file.glb"`

works normally with the example files link extended

**maybe is it something with my FTP configuration? Like security / ports?**

I appreciate any kind of adivice...

**Comment by elalish** [comment]
Are you getting a CORS error or a 404 or something else?

**Comment by renanmalato** [comment]
just a blank file section. the bar sometimes appears like trying to loading but the GLB doesn't shows 

_what I did to fix that, I create a PHP file with the GLB in the same folder and I'm calling the PHP file as an embedded page_

_it's slows me down, but it works._



don't know why the external link doesn't shows the 3D
if you have a moment, give a try on your <model-viewer src=
http://renanmalato.com/stnldealpage/3d/take5jax-reduced10.glb

you can see that the link works for download but doesn't load the GLB

maybe it is something with the FTP ?

**Reply by elalish** [reply]
Have you checked the JS console? This really sounds like a CORS error, which you fix by setting headers server-side.

**Comment by renanmalato** [comment]
Hey @elalish I made some gifs about the problem, look how strange it is...
_by the way thanks so much for your attention_

**You can see here the example Neil Armstrong model works fine, but my GLB link won't**
![gif1](https://github.com/google/model-viewer/assets/3617070/e1edc132-4db7-4f92-963d-fa494dca88c4)


**\You might thinking, so the problem is in the GLB, but it works pretty well on my own domain page.**
![gif2](https://github.com/google/model-viewer/assets/3617070/df6febcf-4bb6-407f-8420-55bd916d609b)


But when I try to use the link on my iframe in another domain (wix) it won't works. Only if I embed my .php file, but in this way I lost a little bit of controls and it slow me down a lot.
![gif3](https://github.com/google/model-viewer/assets/3617070/91764799-9637-4920-822d-fc51b5153873)



I also know that this might be also an WIX trick, but really don't know what is going on.

If you have a moment free try to access I will let for some time with the load link error
https://www.renan-stnl-uxdesign.com/

I'm more designer than dev, so I really don't know how to check the JS console or what is headers server-side

**Comment by renanmalato** [comment]
hey @elalish  I think I found what you are trying to let me know...

<img width="1153" alt="Screen Shot 2024-01-16 at 7 17 04 PM" src="https://github.com/google/model-viewer/assets/3617070/19ebcc22-0193-4ba1-8f96-e12d872dcf95">

is that what you were talking about?

there is a way to unblock that?

I have access to the website head / and i have a main.js linked to the body also

**Reply by elalish** [reply]
Yes, there's another tab or window somewhere called Console - that will give you even more detailed info. Here's our FAQ about [CORS](https://modelviewer.dev/docs/faq.html#entrydocs-technical-questions-cors). I'm afraid you may have to read up a bit on server headers, but I promise it's a very normal thing to have to configure, unless you only serve images.

**Comment by renanmalato** [comment]
Thanks for your time, man.

I changed my course for Sketchfab model viewer for now. 

I tried to contact my server host admin to see if they can activate the SSL to see if this problem will solve. When they get back to me I will try again.

Thanks!
